app/src/utils/formUtilities.test.ts

```diff
@@ -509,3 +510,84 @@ describe('getHridFromValuesAndSpec', () => {
     baseUISpec.viewsets['A'].hridField = undefined;
   });
 });
+
+describe('prettifyFieldName', () => {
+  it('replaces hyphens with spaces', () => {
+    expect(prettifyFieldName('user-name')).toBe('user name');
+    expect(prettifyFieldName('shipping-address-details')).toBe(
+      'shipping address details'
+    );
+  });
+
+  it('removes numeric characters', () => {
```

Comment:
What if the user has put numbers in on purpose? Even if they were added by us because of a duplicate I feel like we should keep them.

Reply:
Okay - I'll update this.